lib/tasks/maintenance.rake

```diff
@@ -0,0 +1,88 @@
+namespace :maintenance do
+  desc "Create full combination flow tests for the Coronavirus Find Support smart answer."
+  task create_find_support_flow_tests: [:environment] do
+    abort "This task will only run in `development`" unless Rails.env.development?
+    abort "Please supply an output_file" if ENV["output_file"].blank?
+
+    filter_options = %i[
+      feeling_unsafe
+      paying_bills
+      getting_food
+      being_unemployed
+      going_to_work
+      somewhere_to_live
+      mental_health
+    ]
+
+    test_counter = 0
+    file = File.open(ENV["output_file"], "w")
```

Comment:
I think it is better to use a block with `File.open` rather than depending on `file.close`. I believe the block version is a little better at dealing with an unexpected halt (as closing occurs no matter how the block is terminated), and it is also less susceptible to the `file.close` being accidentally removed by a later edit. Having said that the garbage collector should sort this out at the end of the task call anyway - so not essential.

So:

```ruby
File.open(ENV["output_file"], "w") do |file|
```
Then on line 85, replace `file.close` with `end`

Reply:
This file is no longer required.